degruyter-numeric.csl

```diff
@@ -0,0 +1,288 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded" default-locale="en-US">
+  <info>
+    <title>De Gruyter (Numeric)</title>
+    <id>http://www.zotero.org/styles/degruyter-numeric</id>
+    <link href="http://www.zotero.org/styles/degruyter-numeric" rel="self"/>
+    <link href="http://www.zotero.org/styles/american-chemical-society" rel="template"/>
+    <author>
```

Comment:
@Aline2024 
Can you add a link to the guidelines here using `rel="documentation"`?